bindings/netstandard/ElectionGuard/ElectionGuard.Encryption/Proofs/SchnorrProof.cs

```diff
@@ -0,0 +1,136 @@
+﻿namespace ElectionGuard.Proofs
+{
+
+    /// <summary>
+    /// Representation of a Schnorr proof
+    /// </summary>
+    public class SchnorrProof : DisposableBase

```

Comment:
moved this into the Encryption library so it can be accessible from everywhere. It can probably move as part of #181